scripts/select_appointments.sql

```diff
@@ -1 +1,18 @@
 -- retrieve appointment information
+select p.surname, p.given_name, a.time
+from patients p
+inner join appointments a
+  on a.patient_id = p.id
+inner join doctors d
+  on a.doctor_id = d.id
+where d.id = 6
```

Comment:
Probably would have done `d.surname = 'Hubbard'` here, since we don't necessarily know where Dr. Hubbard is in the database.